test/migrator-test.js

```diff
@@ -10,50 +10,51 @@ var Migrator = require('../lib/migrator');
 assertDiff.options.strict = true;
 
 describe("Migrator", function() {
-  var tmpPath = "tmp/process-files";
-  var fixturesPath = path.resolve(__dirname, "fixtures");
+  describe('newComponentStructure = flat', function() {
+    var tmpPath = "tmp/process-files";
+    var fixturesPath = path.resolve(__dirname, "fixtures/classic-to-flat");
```

Comment:
I updated the directory name to `fixtures/classic-to-flat`. Everything else is a change in indentation.